lbry/extras/daemon/daemon.py

```diff
@@ -4890,6 +4896,9 @@ async def jsonrpc_blob_list(self, uri=None, stream_hash=None, sd_hash=None, need
         """
         Returns blob hashes. If not given filters, returns all blobs known by the blob manager
 
+        If the filter specify a sd_hash all the blobs for that specific claim are returned if the daemon knows about
```

Comment:
I think this should go further down in the documentation for the options, something like ```--sd_hash=<sd_hash>     : (str) only return blobs in the stream with matching sd hash```

Reply:
Done. I left the "The daemon must know about that sd_hash" because of observed behavior. That is, if trying to list blobs for a stream, it will return empty unless at least the header blob of that stream was downloaded locally.